### PR TITLE
[doc]add spark version

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
#1362

### CODE REVIEW

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md [hunk 1]
```diff
@@ -267,6 +267,7 @@ Example of a mapping file for the input source:
 #### Spark Properties
 
 The following table gives some example properties, all of which can be found in [Spark Available Properties](http://spark.apache.org/docs/latest/configuration.html#available-properties).
+Note: Only Spark > 2.0 version is supported.
```
Comment (jude-zhu) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md]: Not only Spark but also Hadoop, Hive, etc.

Comment (Amber1990Zhang) [reply docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md]: @darionyaphet Please help. What versions do we need here?

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md [hunk 2]
```diff
@@ -267,6 +267,7 @@ Player 表结构如下：
 #### Spark 配置信息
 
 下表给出了一些示例，所有可配置项请见 [Spark Available Properties](http://spark.apache.org/docs/latest/configuration.html#available-properties)。
+注意：为确保 **Nebula Graph** Spark Writer 正常使用，请确保 Spark、Hive、Hadoop 版本至少分别为 2.0、2.3、2.0。
```
Comment (wilsonyou) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md]: 注意- > **注意**
Spark Writer - > **Spark Writer**

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md [hunk 3]
```diff
@@ -267,6 +267,7 @@ Example of a mapping file for the input source:
 #### Spark Properties
 
 The following table gives some example properties, all of which can be found in [Spark Available Properties](http://spark.apache.org/docs/latest/configuration.html#available-properties).
+Note: To use **Nebula Graph** Spark Writer, please make sure your Spark, Hive, Hadoop version are at least 2.0, 2.3, 2.0 respectively.
```
Comment (wilsonyou) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md]: **Note**

and Hadoop versions

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md [hunk 4]
```diff
@@ -267,6 +267,7 @@ Player 表结构如下：
 #### Spark 配置信息
 
 下表给出了一些示例，所有可配置项请见 [Spark Available Properties](http://spark.apache.org/docs/latest/configuration.html#available-properties)。
+**注意：** 为确保 **Nebula Graph Spark Writer** 正常使用，请确保 Spark、Hive、Hadoop 版本至少分别为 2.0、2.3、2.0。
```
Comment (jude-zhu) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md]: "注意" 空行，不然这段文字跟前面文字在同一行

- Spark 2.0
- Hive 2.3
- Hadoop 2.0
p.s. Software Requirements， Prerequisites 一般写在文档的开头

附上现在的效果:

Comment (Amber1990Zhang) [reply docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md]: thanks for taking care of this.

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md [hunk 5]
```diff
@@ -32,9 +40,9 @@ wget https://nebula-graph.oss-accelerate.aliyuncs.com/jar-packages/sst.generator
 This section includes the following steps:
 
 1. Create a graph space and its schema in Nebula Graph
-1. Write data files
-1. Write input source mapping file
-1. Import data
+2. Write data files
```
Comment (whitewum) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md]: why?

Comment (Amber1990Zhang) [reply docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md]: I don't really get your point but this is an ordered list, attached with the preview:
image

**Pull request merged**